### ISSUE TITLE
Ensure that your data is not skewed or biased

### ISSUE THREAD

**Issue by Renger** [Benjamin-Lee/deep-rules#43]
**Have you checked the [list of proposed rules](https://github.com/Benjamin-Lee/deep-rules/issues?q=is%3Aissue+is%3Aopen+label%3Arule) to see if the rule has already been proposed?**

- [x] Yes

**Feel free to elaborate, rant, and/or ramble.**
Training and test sets might contain bias because e.g. more data is available on a subject or specimen. Or a subject has been studied more. Or only positive results are published. This can all result in models with prejudice. It can also ethical issues as described by Cathy O’Neil in the book 'Weapons of Math Destruction'.

**Any citations for the rule?** (peer-reviewed literature preferred but not required)
- [DOI](doi.org/10.5860/crl.78.3.403)

**Comment by Benjamin-Lee** [comment]
Do you mind giving you full name/adding it to [contributors.md](https://github.com/Benjamin-Lee/deep-rules/blob/master/contributors.md)?

Thanks for the contribution!

**Comment by rasbt** [comment]
This is a very important point. However, it is something that affects all data modeling tasks/applications -- i.e., it's not unique to deep learning. I like the idea, but maybe we should all have a discussion at some point deciding whether we want these rules to focus on

a) things that are unique to deep learning (vs trad. machine learning or statistical modeling)
b) having a mix of the two.

Btw regarding the bias/skeweness, the recently released Aequitas project might be relevant: https://dssg.github.io/aequitas/

**Comment by jmschrei** [comment]
I think it would be best to focus on things that are DL specific. That being said, it may also be worth mentioning that many issues you have to be aware of in normal ML models may be particularly important when dealing with DL models, i.e., the potential for overfitting and rigorously tuning hyperparameters.

**Comment by rasbt** [comment]
I agree with you on that. I actually like the rule "Don't forget your ML fundamentals/Rules that apply to ML also apply to DL" #37, and I think we could also include the general issues like "Ensure that your data is not skewed or biased " as a paragraph or a few sentences there.

**Comment by Renger** [comment]
> This is a very important point. However, it is something that affects all data modeling tasks/applications -- i.e., it's not unique to deep learning. I like the idea, but maybe we should all have a discussion at some point deciding whether we want these rules to focus on
> 
> a) things that are unique to deep learning (vs trad. machine learning or statistical modeling)
> b) having a mix of the two.
> 
> Btw regarding the bias/skeweness, the recently released Aequitas project might be relevant: https://dssg.github.io/aequitas/

Agree with the comment that it is not unique for DL. Might be good to note down in the rules which are specific and which are generic. Including the specific rules only might result in readers to overlook the latter.

**Comment by Renger** [comment]
Hi,

I'm rather new to Git so I hope it worked out :-)

Added my name and did a pull request.

Renger.

**Comment by fmaguire** [comment]
Covered in https://github.com/Benjamin-Lee/deep-rules/blob/master/content/03.ml-concepts.md
